src/fields/chips.ts

```diff
@@ -61,7 +61,9 @@ export const chips: IFieldGenerator<IChipsSchema> = (id, field) => {
 				},
 			},
 		};
-		fieldsConfig[`chips-${kebabCase(textarea?.label)}`] = {
+		let textareaId = `${id}-textarea`;
+		if (textarea?.label) textareaId = `${textareaId}-${kebabCase(textarea?.label)}`;
```

Comment:
I'm hesitant on using label because it's a display value, so it's more prone to change vs a static id

Reply:
that's a fair point, decided to remove the label and rely on appending `textarea` to the chips' id
granted there may be a chance of id collision with other fields but I think it's not realistic to prevent that, the onus will be on the dev to ensure this does not happen